v1/linkml-schemas/address.yaml

```diff
@@ -36,15 +37,29 @@ classes:
 
 slots:
   uprn:
+    range: integer
+    slot_uri: adb:uprn
   organisationName:
+    slot_uri: adb:organisationName
   departmentName:
+    slot_uri: adb:departmentName
   subBuildingName:
+    slot_uri: adb:subBuildingName
   buildingNumber:
+    slot_uri: adb:buildingNumber
   buildingName:
+    slot_uri: adb:buildingName
   dependentStreetName:
+    slot_uri: adb:dependentThoroughfare
   streetName:
+    slot_uri: adb:thoroughfare
   doubleDependentAddressLocality:
+    slot_uri: adb:doubleDependentLocality
   dependentAddressLocality:
+    slot_uri: adb:dependentLocality
   addressLocality:
+    slot_uri: adb:postTown
   postalCode:
+    slot_uri: adb:postcode
```

Comment:
Are we in a position while we accept only UK addresses to provide a pattern against the postalCode that we provide?  The addressbase schema has a regular expression defined currently that enforces a space in the middle of the post code.  This would all depend on whether we have similar validation in the user interface should the user decide to overtype a lookup or provide their postcode manually.

Reply:
On the grounds that some users may at some point be allowed to provide a non-UK address for certain purposes, I think it'd be better to avoid bringing in validation of post codes. That doesn't preclude us from linking to addressbase in terms of a slot URI - but in this case I think it might be more clear just to to refer to https://schema.org/postalCode

Reply:
```suggestion
    slot_uri: schema:postalCode
```